_data/header.yml

```diff
@@ -6,6 +6,10 @@
 
 # title: Header title
 
+# Used to show or hide the official usa banner at the top of the site.
+# usa_banner can be 'show' or 'hide'
+usa_banner: show
```

Comment:
Really small nitpick, but I wonder if we should keep these different types of toggles consistent by always using something like `true` / `false`? Don't have strong feelings one way or another, as long as we document what the options are I think that is fine, but just wanted to mention it.

Reply:
Yeah I think choosing `true /false` is the way to go and we've already established that direction in the navigation.yml